### DISCUSSION TITLE
Improve Messaging For User Actions Log

### DISCUSSION THREAD

**Discussion by dautrich** [joomla/joomla-cms#47022] [2. Cold Features]
### Is your feature request related to a problem? Please describe.

On one of my sites, I have set Email Notifications under "User Actions Log Options" of my Super User account to "Yes". Reason: I wanted to get a notification as soon as my deputy has written or changed an article, a module or something else.
![Image](https://github.com/user-attachments/assets/f2ae42c2-b83d-4de8-b4c5-97aa4e3e3485)
Instead, I was flooded with emails when I worked on the website myself. 
IMHO this option should be fine-tuned to get better results.

### Describe the solution you'd like

In addition to the "Events to Email" there should be additional options:

- An option to select the user groups I would like to get notifications for.
- A checkbox "Email self"

### Additional context
The requested information can possibly be found in the User Actions Log by filtering for the requested user. Note that there is no possibility to select for user groups in this dialogue (which might become an additional feature request).
Anyway, I'd prefer an email notification over an active search, as long as I can tune the number of emails to my needs.

**Comment by brianteeman** [comment]
Are you really sure that you want this? Just as you were flooded with emails when you worked on the site you will be flooded by emails when anyone else works on the site.

**Comment by dautrich** [comment]
@brianteeman At least I don't get an email every time I (myself) save an article or a module or ....

**Comment by brianteeman** [comment]
For all the reasons that you dont want to get an email every second when you do something you wont want an email every time a colleague does something. The user action log is not designed for what you want. Workflows is designed for this.

**Comment by dautrich** [comment]
I don't think your approach is correct. IMHO workflow has been built for a totally different use case. 
Let's come back to email notifications. That is an existing feature which -at the time being- is not very useful. My proposal is to make that feature a little bit more efficient with limited cost.

We could improve it for example by setting a time limit of n minutes and collect all the log messages within that timeframe, before sending an email. But this would things much more complicated.

**Comment by brianteeman** [comment]
UAL has certainly not been built for what you are asking

**Comment by alikon** [comment]
@dautrich please check #44640

i would like to move this to discussion

**Comment by Kostelano** [comment]
A related suggestion from me, quite old. To connect the idea

https://github.com/joomla/joomla-cms/issues/34756

**Comment by alikon** [comment]
closed as https://github.com/joomla/joomla-cms/pull/44640 is merged

**Comment by richard67** [comment]
Re-opening as PR #44640 has been reverted. See #44845 for details.
